### ISSUE TITLE
Android emulator setup script doesnt work for android 14 (api 34)

### ISSUE THREAD

**Issue by StanislavMayorov** [ProxymanApp/Proxyman#1916]
## Description

@NghiaTranUIT hello. SSL proxy doesn't works for android 14 emulator (Google API). It works fine for android 13. There's no error in the patch script. 

## Steps to Reproduce

1. Install and run android emulators API 34 (Google API)
<img width="1061" alt="Screenshot 2024-01-19 at 17 05 39" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/00cdac49-ab58-4682-95bf-111ac0c0baf7">
2. Click override emulator

<img width="800" alt="Screenshot 2024-01-19 at 15 27 51" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/85ca5857-3eaa-41cc-a0aa-be372c11ec07">

## Current Behavior
SSL proxy doesn't work.

## Expected Behavior

SSL proxy works.

## Environment

- App version: 4.16.0
- macOS version: macOS 13.6.3

**Comment by NghiaTranUIT** [comment]
@StanislavMayorov can you verify the Proxyman Certificate is installed in Setting app -> Security -> Encryption & Credential -> Trusted Certificate -> User Tab?

May I ask: If you finish the step 5 in the Android docs: https://docs.proxyman.io/debug-devices/android-device

- Add res/xml/network_security_config.xml
- Add to AndroidManifest.xml

to your android app source code?

**Comment by NghiaTranUIT** [comment]
@StanislavMayorov I tested again and it works fine with Android API 34
![Screenshot 2024-01-21 at 09 36 01](https://github.com/ProxymanApp/Proxyman/assets/5878421/ce43204c-f0e3-4f2d-b9d4-aeb7d3df5c28)

<img width="550" alt="Screenshot 2024-01-21 at 09 36 18" src="https://github.com/ProxymanApp/Proxyman/assets/5878421/7bb6c876-4ced-43a1-ae76-bf3486ba6838">

- The important point: You have to complete the 5th Step in the Android Emulator Guide. Otherwise, you will get an SSL Error.
- Here is the android project: https://github.com/ProxymanApp/OKHTTP-Android-Sample

**Comment by StanislavMayorov** [comment]
@NghiaTranUIT thank you for reply. I tested [OKHTTP-Android-Sample](https://github.com/ProxymanApp/OKHTTP-Android-Sample) and reproduced the same error like in my app. 

OKHTTP-Android-Sample Error:
```
Request failed:
java.security.cert.CertPathValidatorException
Trust anchor for certification path not found
```

<img width="591" alt="Screenshot 2024-01-21 at 18 17 54" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/98bda48b-b2f7-4845-b595-2e5dcf81619c">
<img width="522" alt="Screenshot 2024-01-21 at 17 43 48" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/9c4a22e4-d510-4c8e-bf12-96450e69a8e2">
<img width="491" alt="Screenshot 2024-01-21 at 18 18 52" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/2dc7557c-e5cb-4bc6-88e8-63544768ef34">

<img width="1447" alt="Screenshot 2024-01-21 at 18 13 18" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/c458360a-7312-4fdb-8283-ba8e73053f8f">

**Comment by NghiaTranUIT** [comment]
It's odd. Let me try again with your emulator (Created by Other Image Tab -> Upsidedown Cake, API 34, Google APIs).

Mime was created from the Recommended Tab.

**Comment by NghiaTranUIT** [comment]
@StanislavMayorov it just works fine again with the sample code, that I provide you.

There is something odd, the first request always fails. If you re-send the GET request again, it will work.

**Comment by StanislavMayorov** [comment]
@NghiaTranUIT I got it. I can reproduce it only on the new `Medium phone` template! It works fine on `pixel 7` with the same API SDK. I have android studio 2023.1.1. `Medium phone` template is a new one.
<img width="638" alt="Screenshot 2024-01-26 at 15 34 57" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/85385140-b8db-49d9-8295-fc1a99bfb316">
<img width="1095" alt="Screenshot 2024-01-26 at 15 35 30" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/792e74c0-e3b8-4b7f-9217-fc56a834c8d3">

**Comment by NghiaTranUIT** [comment]
@StanislavMayorov 

Medium Phone doesn't work with Proxyman because it's a PlayStore version. Pixel phone works fine because it's not 👍 

![Screenshot 2024-01-27 at 10 21 56](https://github.com/ProxymanApp/Proxyman/assets/5878421/e31b9297-d2d2-4c30-9332-aa37414a6a30)

**Comment by StanislavMayorov** [comment]
@NghiaTranUIT pixel 7 has the same play store icon, but it works fine with Google API image.

![image](https://github.com/ProxymanApp/Proxyman/assets/11670429/3fdd8881-fa99-4b1d-a195-82466d694ecc)

**Comment by StanislavMayorov** [comment]
@NghiaTranUIT Hello. Pixel 7 and Medium Phone have the same play store icon. Could you explain the difference please?

![300149741-e31b9297-d2d2-4c30-9332-aa37414a6a30](https://github.com/ProxymanApp/Proxyman/assets/11670429/4e254f8e-575a-404d-9b11-30cf3593f081)

**Comment by StanislavMayorov** [comment]
@NghiaTranUIT I can use `pixel` instead of `Medium Phone`. Pixel works fine with the same API. I hadn't just known about this compatibility issue. It had been confusing.
Can you update this comment to describe that only pixel emulator is supported please?
<img width="749" alt="Screenshot 2024-02-07 at 12 31 10" src="https://github.com/ProxymanApp/Proxyman/assets/11670429/12c5a60a-b0d9-4d9c-b7b3-7885eb1bccaf">

**Comment by NghiaTranUIT** [comment]
Sure, I will add this note. For now, I might update the online documentation first 👍

**Comment by StanislavMayorov** [comment]
thank you!
